x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -131,6 +131,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     describe('SearchBar', () => {
       it('add filter', async () => {
         await filterBar.addFilter({ field: 'rule.name', operation: 'is', value: ruleName1 });
+        sleep(1000);
```

Comment:
Instead of the following sleep we should call `HeaderPageObject.awaitGlobalLoadingIndicatorHidden` on before test